test/integration/error_output.cc

```diff
@@ -146,3 +145,30 @@ TEST(Error, ErrorOutput)
   // Check nothing has been printed
   EXPECT_TRUE(buffer.str().empty()) << buffer.str();
 }
+
+////////////////////////////////////////
+// Test PrintConfig class for sdf::Errors outputs
+TEST(ErrorOutput, PrintConfigErrorOutput)
+{
+  std::stringstream buffer;
+  sdf::testing::RedirectConsoleStream redir(
+    sdf::Console::Instance()->GetMsgStream(), &buffer);
+
+  sdf::Errors errors;
+  sdf::PrintConfig printConfig;
+  ASSERT_FALSE(printConfig.SetRotationSnapToDegrees(361, 300, errors));
+  EXPECT_NE(std::string::npos, errors[0].Message().find(
+    "Interval value to snap to must be larger than 0,"
+    " and less than or equal to 360."));
+  ASSERT_EQ(errors.size(), 1u);
```

Comment:
This assertion should come before indexing into `errors` to avoid a crash when `errors` is empty.

Reply:
Fixed in https://github.com/gazebosim/sdformat/pull/1098/commits/a498d9e371ab03a6a4a9b59ef66869eb7369b06d.